packages/components/src/components/date-picker/date-picker.scss

```diff
@@ -102,10 +102,6 @@
             min-width: calc(100% - 1rem);
             position: absolute;
             height: 38px;
-
```

Comment:
Is `disabled` class modifier no longer used?

Reply:
Seems like this particular style did not do anything as border-coloris modified in other css